tests/test_auto_provisioning.py

```
@@ -0,0 +1,72 @@
+    
+    # 2. Instantiate broker (should trigger auto-download)
+    print("\n🎯 Creating AngelOne instance...")
+    print("   (This should automatically download master contract)\n")
+    
+    from india_stocks_api.brokers import AngelOne
+    
+    # Use dummy credentials - we're only testing DB provisioning, not authentication
+    broker = AngelOne(
+        api_key="DUMMY_KEY",
+        client_code="DUMMY_CODE",
+        password="DUMMY_PIN",
```

Ruff (0.14.13)

Possible hardcoded password assigned to argument: "password"

(S106)